scripts/check_frontend_test_coverage.py

```diff
@@ -317,8 +317,8 @@
     'teacher.ts',
     'teacher2.ts',
     'thumbnail-uploader.directive.ts',
+    'top-navigation-bar.component.ts',
     'top-navigation-bar-wrapper.component.ts',
```

Comment:
Remove the wrapper component from the codebase.

Reply:
And make side navigation bar component directly read the isSidebarShown status from the service itself, instead of passing it as a @Input. If that is successful, remove the side-navigation-bar-wrapper component too.

Reply:
done